li-utils/src/main/javaPegasus/com/linkedin/common/urn/Urn.java

```diff
@@ -83,6 +85,13 @@ public Urn(String rawUrn) throws URISyntaxException {
           0);
     }
 
+    if (rawUrn.length() > NUM_BYTES_LIMIT) {
```

Comment:
This constructor doesn't cover all use cases for urn validation, just the from string ones. Is this also intended to cover the URL Encoding increased length that can occur? Don't see a call to convert to URL encoded to verify length